documentation/modules/creating-storage-mapping.adoc

```diff
@@ -6,24 +6,29 @@
 [id="creating-storage-mapping_{context}"]
 = Creating a storage mapping
 
-You can create a storage mapping by using the {ocp} web console to map source data stores to {virt} storage classes.
+You can create a storage mapping by using the {ocp} web console to map source storage domains to {virt} storage classes.
```

Comment:
`storage domains` is specific to ovirt/rhv, this may confused users that migrate from vsphere
maybe it would be better to use something more general here, like `disk storage`